corehq/motech/forms.py

```diff
@@ -46,10 +46,15 @@ class Meta:
 
     def __init__(self, *args, domain, **kwargs):
         self.domain = domain  # Passed by ``FormSet.form_kwargs``
+        instance = kwargs.get('instance', None)
+        if instance:
+            kwargs['initial'] = {'plaintext_password': instance.plaintext_password, }
```

Comment:
I am not sure when this would happen, but the parent class allows both `instance` and `initial` to be set at the same time. Instead of overwriting `initial`, it might be best to update it if it is already set
```suggestion
            if 'initial' in kwargs:
                kwargs['initial']['plaintext_password'] = instance.plaintext_password
            else:
                kwargs['initial'] = {'plaintext_password': instance.plaintext_password}
```

Reply:
Also, could you send `PASSWORD_PLACEHOLDER` to the browser instead of the real password? It may be an unnecessary precaution, but I think it's better than telling the user what the password is. (I mean, they shouldn't need us to tell them, right?) The setter already only changes the password if it isn't the placeholder:
https://github.com/dimagi/commcare-hq/blob/6646bda0727e8bc2f8b6eb6ccc5725f7d034e289/corehq/motech/models.py#L54-L60